src/main/java/com/google/sps/COMMONS.java

```diff
@@ -19,9 +19,9 @@
 
 /** Definition of constants that servlets will use. */
 public interface COMMONS {
-  float MAX_DISTANCE_CLIENT = 20000f; // 20 kilometers
-  float MAX_DISTANCE_VENDOR = 3000f; // 3 kilometers
-  float MIN_DISTANCE = 0f;
+  int MAX_DISTANCE_CLIENT = 20000; // 20 kilometers
```

Comment:
Nit: I'd consider doing this to make it easier to read (here and below)
```suggestion
  int MAX_DISTANCE_CLIENT = 20 * 1000; // 20 kilometers
```

Reply:
Great advice, thank you.